src/components/hv-date-field/index.tsx

```diff
@@ -38,13 +40,11 @@ export default class HvDateField extends PureComponent<HvComponentProps> {
     return getNameValueFormInputValues(element);
   };
 
-  props: HvComponentProps;
-
   /**
    * Given a Date object, returns an ISO date string (YYYY-MM-DD). If the Date
    * object is null, returns an empty string.
    */
-  static createStringFromDate = (date: ?Date): string => {
+  static createStringFromDate = (date?: Date | null): string => {
```

Comment:
suspicious - can we make the argument presence non optional (like the original signature)

Reply:
57970c0